routes/web.php

```diff
@@ -129,8 +134,13 @@
 });
 
 // TODO: remove devlopment routes
+// TODO: do it next year
```

Comment:
Dev routes in production is really unsave. Please consider removing them in version 2.1.X

Reply:
Maybe